docs/guides/creatingadls.md

```diff
@@ -0,0 +1,140 @@
+      ref:
+        apiVersion: serving.knative.dev/v1
+        kind: Service
+        name: example-sink
+    backoffDelay: <duration>
+    backoffPolicy: <policy-type>
+    retry: <integer>
+```
+
+### Broker Configuration
+
+You can configure how events are delivered for each Broker by adding a delivery spec, as shown in the following example:
+
+```yaml
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  name: with-dead-letter-sink
+spec:
+  delivery:
+    deadLetterSink:
+      ref:
+        apiVersion: serving.knative.dev/v1
+        kind: Service
+        name: example-sink
+    backoffDelay: <duration>
+    backoffPolicy: <policy-type>
+    retry: <integer>
+```
+
+### Example Broker Configuration
+
+Lets take a look at an example of a Broker with a dead letter sink and configure a simple bridge with a dead letter sink. For our discussion, let us consider [this example Bridge](../assets/yamlexamples/simple-bridge.yaml) as a starting point:
+
+```yaml
+apiVersion: flow.triggermesh.io/v1alpha1
+kind: Bridge
+metadata:
+  name: cron-sockeye
+  annotations:
+    bridges.triggermesh.io/name: cron-sockeye
+    bridges.triggermesh.io/description: >-
+      This is a simple starter bridge containg only a PingSource and a broker.
+spec:
+  components:
+    - object:
+        apiVersion: eventing.knative.dev/v1
+        kind: Broker
+        metadata:
+          name: events
+      options:
+        enableResync: false
+    - object:
+      apiVersion: sources.knative.dev/v1
+      kind: PingSource
+      metadata:
+        name: ping-sockeye
+      spec:
+        data: '{"name": "triggermesh"}'
+        schedule: "*/1 * * * *"
+        sink:
+          ref:
+            apiVersion: eventing.knative.dev/v1
+            kind: Broker
+            name: events
+            options:
+              enableResync: false
+```
+
+Now, lets modify this example to add a dead letter sink to the `Broker`. We can accomplish this in two steps:
+
+1. Add a service to catch the "dead letter" events. (We will be using `Sockeye` here, but in a production scenario you would want to use something like SQS, Kafka, or another Sink that has some form of persistence.)
+
+```yaml
+    - object:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      metadata:
+        name: sockeye
+      spec:
+        template:
+          spec:
+            containers:
+              - image: docker.io/n3wscott/sockeye:v0.7.0@sha256:e603d8494eeacce966e57f8f508e4c4f6bebc71d095e3f5a0a1abaf42c5f0e48
+```
+
+2. Update the `Broker` section to the following:
+
+```yaml
+    - object:
+    apiVersion: eventing.knative.dev/v1
+    kind: Broker
+    metadata:
+    name: events
+    spec:
+    delivery:
+        deadLetterSink:
+        ref:
+            apiVersion: serving.knative.dev/v1
+            kind: Service
+            name: sockeye
+        backoffDelay: "PT0.5S"     # or ISO8601 duration
+        backoffPolicy: exponential # or linear
+        retry: 2
+```
+
```

Comment:
Does this Bridge design essentially cause errors to occur which are then sent to the DLS? (sorry am being lazy and have not executed the guide). If so, maybe we could explain that here, e.g. "The design of this Bridge is such that the backoffDelay and pingsource frequency cause events to be rejected and sent to the dead letter sink"

Reply:
no.. the bridge should function as expected here. I suppose I should include a section on breaking the bridge to induce events into the DLS?

Reply:
I think we're ok for now!